src/lib.rs

```diff
@@ -134,7 +134,7 @@ impl std::default::Default for Assert {
         Assert {
             cmd: vec!["cargo", "run", "--"]
                 .into_iter().map(String::from).collect(),
-            expect_success: true,
+            expect_success: None,
```

Comment:
yes _and_ that would fix a documentation bug, I introduced. I wrote everywhere `/// Defaults to asserting _successful_ execution.` ^^

Adding it to the todo list.